griddedio/GriddedIO.F90

```diff
@@ -128,6 +129,11 @@ subroutine CreateFileMetaData(this,items,bundle,timeInfo,vdata,ogrid,global_attr
         character(len=:), pointer :: attr_name, attr_val
         integer :: status
 
+        if ( allocated (this%metadata) ) deallocate(this%metadata)
+        allocate(this%metadata)
+
+        call MAPL_FieldBundleDestroy(this%output_bundle, _RC)
+
         this%items = items
         this%input_bundle = bundle
         this%output_bundle = ESMF_FieldBundleCreate(rc=status)
```

Comment:
@bena-nasa: I was concerned about the added L135, until I see the L139.  So that is safe (first destroy bundle, then create bundle). It is easier to see the change in context on github than from the terminal. 
Meanwhile a question is what if this%output_bundle does not exist in the first hand, 
will  call MAPL_FieldBundleDestroy(this%output_bundle, _RC)  lead to an error?   
Maybe a safer way is  if (present (output_bundle)) destroy output_bundle.  Does ESMF handle this gracefully?

Reply:
`output_bundle` is not an optional argument, so we cannot use `present(output_bundle)`.

But, yes, calling destroy on an uninitialized object should lead to an error condition.   I'm not 100% confident that ESMF correctly does this.  It would assume they have a derived type component with default initialization that they can use to check things.  (As oppossed to just assuming untouched memory has the value they expect.)